### ISSUE TITLE
One line break answers both `declaration-colon-newline-after` and `declaration-block-semicolon-newline-before`, and the configuration's order decides where it lands

### ISSUE THREAD

**Issue by firefoxic** [stylelint-stylistic/stylelint-stylistic#417]
### What minimal example or steps are needed to reproduce the bug?

A fifth member of [#356](https://github.com/stylelint-stylistic/stylelint-stylistic/issues/356), and none of the four classes that umbrella names holds it.

`declaration-colon-newline-after` with `always` wants a line break behind the colon. `declaration-block-semicolon-newline-before` with `always` wants one in front of the semicolon. Where the declaration's value holds no word of its own — where all that stands behind the colon is whitespace, comments and a flag — the same one break answers both of them, and whichever rule runs first is the one that writes it:

```css
a { color:  /*c*/ ; }
```

| order | `--fix` |
| --- | --- |
| the semicolon rule first | `a { color:  /*c*/\n; }` |
| the colon rule first | `a { color:  /*c*/\n ; }` |

Both files are ones the pair is content with, so no warning is left unsaid; the output is simply not the configuration's to predict. What the order decides is whether the space standing in front of the semicolon survives the break: the semicolon rule takes it with it when it runs first, and leaves it where it stands when it runs second.

### What minimal configuration is needed to reproduce the bug?

```json
{
	"plugins": [
		"@stylistic/stylelint-plugin"
	],
	"rules": {
		"@stylistic/declaration-colon-newline-after": "always",
		"@stylistic/declaration-block-semicolon-newline-before": "always"
	}
}
```

and the same two entries the other way round.

### What did you expect to happen?

One file, whichever order the configuration spells them in.

### What actually happened?

Two, and which one the user is left with is decided by the order two entries happen to stand in.

### Notes

It is none of the four: neither rule takes a `-single-line` or `-multi-line` option, so it is not [#355](https://github.com/stylelint-stylistic/stylelint-stylistic/issues/355); neither is `linebreaks` or `indentation`, so it is not [#352](https://github.com/stylelint-stylistic/stylelint-stylistic/issues/352) or [#353](https://github.com/stylelint-stylistic/stylelint-stylistic/issues/353); and no trailing semicolon is written, so it is not [#354](https://github.com/stylelint-stylistic/stylelint-stylistic/issues/354). What it is instead is two rules asked about one stretch of text from either end, each satisfied by the same character, and the one that runs first deciding how much of the run beside it goes with that character.

`pairs.mjs` reports it as a `soft` row, and only from 2026-08-27, when the `wordless-value` fixture joined its corpus with the branch of [#371](https://github.com/stylelint-stylistic/stylelint-stylistic/issues/371) — the corpus held no declaration whose value has no word until then. The row stands on that branch and not on `main`, and only because on `main` `declaration-colon-newline-after` never converged on that shape at all, so every pair it stood in was dropped as a configuration contradicting itself.
